osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs

```diff
@@ -60,7 +68,7 @@ private void load(OsuConfigManager osuConfig, FrameworkConfigManager config)
             if (RuntimeInfo.OS != RuntimeInfo.Platform.Windows)
             {
                 rawInputToggle.Disabled = true;
-                sensitivity.Bindable.Disabled = true;
+                sensitivityBindable.Disabled = true;
```

Comment:
I was confused as to why this was the only line changed when the original report came from Windows (and why I couldn't repro it), but then I pulled up the entire file and noticed this lower down:

https://github.com/ppy/osu/blob/aec2520ef41155ea14e7bdf0c68fce4a45c7e3e1/osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs#L89

so I guess what I'd like to propose in a long-winded way is to either revert this change or change both places? It's the same reference so it doesn't really matter either way, but probably better to keep consistent.

Reply:
Nice catch, completely missed this one.